Leaderboard/package.json

```diff
@@ -3,21 +3,32 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www",
-    "test": "mocha"
+    "start": "DEBUG=Leaderboard nodemon ./bin/www",
+    "test": "mocha --compilers js:babel/register"
   },
   "dependencies": {
+    "babel": "^5.8.23",
     "body-parser": "~1.13.2",
+    "config": "^1.16.0",
+    "connect-session-sequelize": "^2.3.1",
     "cookie-parser": "~1.3.5",
     "debug": "~2.2.0",
     "express": "~4.13.1",
+    "express-session": "^1.11.3",
+    "express-winston": "^0.4.1",
     "hbs": "~3.1.0",
+    "js-yaml": "^3.4.2",
+    "lodash": "^3.10.1",
     "morgan": "~1.6.1",
-    "serve-favicon": "~2.3.0"
+    "nodemon": "^1.6.1",
+    "sequelize": "^3.9.0",
+    "serve-favicon": "~2.3.0",
+    "winston": "^1.0.2"
   },
   "devDependencies": {
     "chai": "^3.2.0",
     "mocha": "^2.3.2",
+    "sqlite3": "^3.1.0",
```

Comment:
In production, we will be using postgres, so this is why it's under dev dependencies.